src/client/api/mgmt.c

```diff
@@ -112,24 +124,82 @@ daos_pool_create(uint32_t mode, uid_t uid, gid_t gid, const char *grp,
 		return -DER_INVAL;
 	}
 
+	entries = (pool_prop == NULL) ? 0 : pool_prop->dpp_nr;
+
+	/*
+	 * TODO: remove uid/gid input params and use euid/egid instead
```

Comment:
I think its ok to use this like this, You may want to create a pool with a uid/gid not yours. In the current dmg tool they make the decision whether to pull euid/egid or to use what is passed in on the command line.

Reply:
In that case the owner user/group should be passed in as part of the pool_prop, as strings. It's only if nothing is passed in to us that we should map it to the euid/egid. I believe this entry point actually goes away altogether once the new DMG tool is done, so the TODO applies only to the short-term usage of this API.